src/test/java/seedu/address/testutil/EditPropertyDescriptorBuilder.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.testutil;
+
+import seedu.address.logic.commands.EditPropertyCommand.EditPropertyDescriptor;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.name.Name;
+import seedu.address.model.property.Address;
+import seedu.address.model.property.PostalCode;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.Type;
+
+/**
+ * A utility class to help with building EditPropertyDescriptor objects.
+ */
+public class EditPropertyDescriptorBuilder {
+
+    private EditPropertyDescriptor descriptor;
+
+    public EditPropertyDescriptorBuilder() {
+        descriptor = new EditPropertyDescriptor();
+    }
+
+    public EditPropertyDescriptorBuilder(EditPropertyDescriptor descriptor) {
+        this.descriptor = new EditPropertyDescriptor(descriptor);
+    }
+
+    /**
+     * Returns an {@code EditPropertyDescriptor} with fields containing {@code property}'s details
+     */
+    public EditPropertyDescriptorBuilder(Property property) {
+        descriptor = new EditPropertyDescriptor();
+        descriptor.setName(property.getName());
+        descriptor.setPostalCode(property.getPostalCode());
+        descriptor.setDeadline(property.getDeadline());
+        descriptor.setAddress(property.getAddress());
+        descriptor.setType(property.getPropertyType());
+    }
+
+    /**
+     * Sets the {@code Name} of the {@code EditPropertyDescriptor} that we are building.
+     */
+    public EditPropertyDescriptorBuilder withName(String name) {
+        descriptor.setName(new Name(name));
+        return this;
+    }
+
+    /**
+     * Sets the {@code PostalCode} of the {@code EditPropertyDescriptor} that we are building.
+     */
+    public EditPropertyDescriptorBuilder withPostalCode(String postal) {
+        descriptor.setPostalCode(new PostalCode(postal));
+        return this;
+    }
+
+    /**
+     * Sets the {@code Deadline} of the {@code EditPropertyDescriptor} that we are building.
+     */
+    public EditPropertyDescriptorBuilder withDeadline(String deadline) {
+        try {
+            descriptor.setDeadline(ParserUtil.parsePropertyDeadline(deadline));
+            return this;
+        } catch (ParseException ex) {
+            return this;
+        }
+    }
+
+    /**
+     * Sets the {@code Address} of the {@code EditPropertyDescriptor} that we are building.
+     */
+    public EditPropertyDescriptorBuilder withAddress(String address) {
+        descriptor.setAddress(new Address(address));
+        return this;
+    }
+
+    /**
+     * Parses the {@code tags} into a {@code Set<Tag>} and set it to the {@code EditPropertyDescriptor}
+     * that we are building.
+     */
```

Comment:
Rmb to update this javadoc

Reply:
Thanks for catching this! should be fixed!